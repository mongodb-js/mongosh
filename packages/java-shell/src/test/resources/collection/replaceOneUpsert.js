// before
db.coll.remove({});
db.coll.insertOne({a: 1});
// command
db.coll.replaceOne({a: 2}, {a: 3}, {upsert: true});
// command
db.coll.find();
