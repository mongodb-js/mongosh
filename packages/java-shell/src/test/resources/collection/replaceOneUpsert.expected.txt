AcknowledgedUpdateResult{matchedCount=0, modifiedCount=0, upsertedId=null}
[ { "_id": <ObjectID>, "a": 1 }, { "_id": <ObjectID>, "a": 3 } ]